<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
background-color: rgb(18, 18, 18);
border: 2px solid rgb(239, 205, 129);
}

QPushButton {
background-color: rgb(237, 201, 125);
}

QPushButton:hover {
background-color: rgb(247, 222, 153);
}

QToolButton {
background-color: rgb(237, 201, 125);
}

QToolButton:hover {
background-color: rgb(247, 222, 153);
}</string>
  </property>
  <widget class="QPushButton" name="BotonFichaje">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Cambria&quot;;

</string>
   </property>
   <property name="text">
    <string>EMITIR FICHAJE</string>
   </property>
  </widget>
  <widget class="QLabel" name="FechaActual">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
font-size: 30px;
text-align: center;
font-weight: bold;
border: 0px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HoraActual">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
font-size: 30px;
text-align: center;
font-weight: bold;
border: 0px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="BotonVolver">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
</string>
   </property>
   <property name="text">
    <string>тна</string>
   </property>
  </widget>
  <widget class="QLabel" name="DatosCliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
text-align: center;
font-size: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BotonFichajeSalida">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Cambria&quot;;

</string>
   </property>
   <property name="text">
    <string>FICHAR SALIDA</string>
   </property>
  </widget>
  <widget class="QLabel" name="DatosClienteEntrada">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
text-align: center;
font-size: 15px;
border: 0px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DatosClienteSalida">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(239, 205, 129);
text-align: center;
font-size: 15px;
border: 0px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
